### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by pypingou** [Guake/guake#552]
guake-0.6.0-1.fc22

Truncated backtrace:
prefs.py:597:on_palette_name_changed:IndexError: list index out of range

Traceback (most recent call last):
```
  File "/usr/lib64/python2.7/site-packages/guake/prefs.py", line 597, in on_palette_name_changed
    PALETTES[palette_index])
IndexError: list index out of range
```

Local variables in innermost frame:
palette_index: 9
self: PrefsDialog(path="/usr/share/guake/prefs.glade", root="config_window")
combo: <gtk.ComboBox object at 0x7f201d2a2780 (GtkComboBox at 0x1d8d3d0)>

I guess this is related to the different palette introduced in 0.6+, are we careful with how we set them?

Maybe we should use ``PALETTES.get(palette_index, <default>)``

**Comment by pypingou** [comment]
Original report: https://bugzilla.redhat.com/1223122

**Comment by Stibbons** [comment]
Should have been fixed in 0.7.x.

**Comment by pypingou** [comment]
>    Should have been fixed in 0.7.x.

Ok thanks!

I have not updated Fedora to 0.7.x yet seeing the number of reports we have on
them atm, but I will.
